lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
```

Comment:
Your player object appears to not have any useful task aside from instantiating itself. I suggest putting some logic here that you placed in the Game class - logic related to player.